src/GitHub.Extensions/TaskExtensions.cs

```diff
@@ -20,5 +20,8 @@ public static class TaskExtensions
                 return default(T);
             }
         }
+        public static void Forget(this Task task)
```

Comment:
I keep thinking `Tasks` are "cold" (aka lazy) when they're not, which is why I was confused that this would actually work. This name is appropriate.